pegasus/sites.v3/code.org/views/music/music_lab_student_projects.haml

```diff
@@ -0,0 +1,21 @@
+-# FontAwesome icons
+- icon_check = "fa-solid fa-check-circle"
+
+:ruby
+  list_item = [hoc_s("music_lab.student_projects.list_create"), hoc_s("music_lab.student_projects.list_artist"),  hoc_s("music_lab.student_projects.list_share")]
+
+.grid-container.gap-2
+  %iframe.rounded-corners.box-shadow-primary{width: "100%", height: 300, src: CDO.studio_url("/musiclab/embed?channels=#{request.params['channels']}&library=#{request.params['library']}", CDO.default_scheme)}
```

Comment:
@breville here's where this is embedded — it's not showing up for me locally (see screenshots) so let me know if I need to replace anything!

Reply:
hi Kelby, the channel IDs are environment-specific, so if you are testing locally, it means you need to create a new music project locally first.  Given that this page will be unlinked for a while, I think we can go ahead and test this out on production when it makes it to there.

Reply:
nit: you could probably also extract the `src` string into a Ruby variable like you have for `list_item`.

Reply:
Ok cool! 🙌  

And what's the reasoning for extracting the `src` string? Does it have a functional benefit or just makes the code cleaner?